### PR TITLE
Add ZMQ port for orchagent

### PR DESCRIPTION
**Pull request by liuh-80**
Add ZMQ port for orchagent.

#### Work item tracking
Microsoft ADO (number only): 17753804

#### Why I did it
Orchagent need create ZMQ server for listen incoming connection.

#### How I did it
Add ZMQ address to zmqserver.h

#### How to verify it
Pass all UT and E2E test cases.

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111

#### Description for the changelog
Add ZMQ port for orchagent.

### CODE REVIEW

**Comment by liuh-80** [comment]
Move to ZMQ .h

File: common/zmqserver.h [hunk 1]
```diff
@@ -11,6 +11,9 @@
 #define MQ_POLL_TIMEOUT (1000)
 #define MQ_WATERMARK 10000
 
+/***** ZMQ PORT *****/
+#define DASH_ORCH_ZMQ_PORT                 1234
+
```
Comment (Pterosaur) [inline common/zmqserver.h:16]: Will this value be accessed by SWIG interface? 
In my mind, other components may use this value via SWIG of Python or Golang.

File: common/zmqserver.h [hunk 2]
```diff
@@ -11,6 +11,9 @@
 #define MQ_POLL_TIMEOUT (1000)
 #define MQ_WATERMARK 10000
 
+/***** ZMQ PORT *****/
+#define DASH_ORCH_ZMQ_PORT                 1234
```
Comment (qiluo-msft) [inline common/zmqserver.h:15]: >1234 [](http://example.com/codeflow?start=43&length=4)

Let's pick one not used large port number. ref: [https://en.wikipedia.org/wiki/List_of_TCP_and_UDP_port_numbers](https://en.wikipedia.org/wiki/List_of_TCP_and_UDP_port_numbers)

Comment (qiluo-msft) [inline common/zmqserver.h:15]: >DASH_ORCH_ZMQ_PORT [](http://example.com/codeflow?start=8&length=18)

-> ZMQ_SERVER_PORT_DEFAULT
1. No necessary only DASH
2. No necessary only used in orchagent

**Comment by qiluo-msft** [comment]
The PR title and PR description suggestion:
ZMQ address -> ZMQ port

File: common/zmqserver.h (hunk 1 above)
Comment (liuh-80) [reply common/zmqserver.h]: Fixed, change to static const for SWIG export it to wrapper.

File: common/zmqserver.h (hunk 2 above)
Comment (liuh-80) [reply common/zmqserver.h]: Fixed, change to 8020

Comment (liuh-80) [reply common/zmqserver.h]: Change to ORCH_ZMQ_PORT
ZMQ port can't be reuse, which means if orchagent bind to 8020 port, other app can not bind to it again.
So ORCH_ZMQ_PORT is only for orchagent, other app can't reuse it.

**Review by qiluo-msft — APPROVED** [review]

**Review by Pterosaur — APPROVED** [review]

**Pull request merged**